virtio-devices/Cargo.toml

```diff
@@ -7,6 +7,7 @@ edition = "2018"
 [features]
 default = []
 io_uring = ["block_util/io_uring"]
+mshv = ["hypervisor/mshv"]
```

Comment:
As we only need a feature gate, let's use the `mshv = []` here so that we don't need to add the unused dependency on the `hypervisor` crate.